lock-conjure-api/src/main/conjure/lock-api.yml

```diff
@@ -44,18 +46,24 @@ services:
       lockAndGetHeldLocks:
         http: POST /laghl/{namespace}
         args:
-          namespace: string
+          namespace:
```

Comment:
`internalTableName` are unsafe but `namespaces` are safe -- just confirming that is ok

Reply:
yeah. `namespace` is almost always declared as part of a user's atlas config, but the table names are in some contexts generated at runtime